packages/template-retail-react-app/app/theme/components/project/product-view.js

```diff
@@ -0,0 +1,12 @@
+import {ProductView} from 'pwa-kit-ecom/theme/components'
+
+export default {
+    ...ProductView,
+    baseStyle: {
+        ...ProductView.baseStyle,
+        container: {
+            ...ProductView.baseStyle.container,
+            background: 'yellow.100'
+        }
+    }
```

Comment:
I think it would be better if the developer can simply write only the change that they wanted like this:

```js
export default {
    baseStyle: {
        container: {
            background: 'yellow.100'
        }
    }
```

It is possible if we call the `extendTheme` like what I mentioned in the previous comment.